scripts/e2e-tests.sh

```diff
@@ -144,37 +148,37 @@ function delete_leaked_network_resources() {
   fi
 }
 
-# Create a test cluster with kubetest and call the current script again.
+# Check ${CLUSTER_CREATION_LOG} to see if it contains key words for stockout
+function cluster_creation_stockout() {
+  grep -Eio "does not have enough resources available to fulfill the request" "${CLUSTER_CREATION_LOG}">/dev/null 2>&1
```

Comment:
These key words are quite a few hence this check will be easily impacted by the log change.
Is there a more reliable way for asserting stockout?

Reply:
NVM, seems it's already been discussed below.